README.md

```diff
@@ -27,6 +27,9 @@ An example `Brewfile`:
 tap "homebrew/cask"
 # 'brew tap' with custom Git URL
 tap "user/tap-repo", "https://user@bitbucket.org/user/homebrew-tap-repo.git"
+# 'brew tap' with arguments
+tap "user/tap-repo", "https://user@bitbucket.org/user/homebrew-tap-repo.git", args: ["force-auto-update"]
```

Comment:
```suggestion
tap "user/tap-repo", "https://user@bitbucket.org/user/homebrew-tap-repo.git", force_auto_update: true
```

What would you think about an API like this? Are there other arguments you need?

Reply:
Can do. Would you prefer if we *only* support `--force-auto-update`, or allow both 
```
tap "user/tap-repo", "https://user@bitbucket.org/user/homebrew-tap-repo.git", force_auto_update: true
```

as well as

```
tap "user/tap-repo", "https://user@bitbucket.org/user/homebrew-tap-repo.git", args: ["arbitrary-argument"]
```

It is trivial to support both, as well as

```
tap "user/tap-repo", "https://user@bitbucket.org/user/homebrew-tap-repo.git", args: {arg1: true, "arg-two": "value"}
```

which would correspond to

```
--arg1 --arg-two=value
```

I will go ahead and push a version the does that. If folks prefer, I can delete all that and just support `force_auto_update`.

Reply:
> Would you prefer if we _only_ support `--force-auto-update`

Yeh, only support I think.

Reply:
Done!